enterprise-suite/tests/smoketest_minikube.sh

```diff
@@ -22,6 +22,8 @@ function diagnostics() {
 }
 
 function setup() {
+	local es_console_version; es_console_version=''
+	[[ -n "${ES_CONSOLE_VERSION:-}" ]] && es_console_version=",esConsoleVersion=${ES_CONSOLE_VERSION}"
```

Comment:
It would be nicer to just do `es_console_version="--set esConsoleVersion=${ES_CONSOLE_VERSION}"`.